### DISCUSSION TITLE
Use HTML href tags instead of JS onclick redirect for clickable rows.

### DISCUSSION THREAD

**Discussion by siebsie23** [rappasoft/laravel-livewire-tables#939] [Feature Requests]
Currently the clickable rows use the onclick= attribute on table rows.
Because of this, it's not possible to open rows in a new tab by right clicking or middle mouse button etc.

**Comment by jurri** [comment]
Its a table not an anchor-link.

It uses the onclick-method on every Table-Data (td). It is relevant because u can remove it on every Column:  `Column::make("Name")->unclickable()`

Add `setTableRorUrlTarget()` for returning '_blank' to open in new tab.

[see here how to use](https://rappasoft.com/docs/laravel-livewire-tables/v2/rows/clickable-rows)
---

For editing Table-Row use `setTrAttributes()`. U can use it for adding `onclick` or `wire:click` attributes.
https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods#content-settrattributes

use action buttons for your need.
